src/pages/experimental/create-flows/index.mdx

```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
+If there are only one or two options to be made during a creation, use a small, standard modal overlaying the content where the creation was initiated. Avoid scrolling inside a small modal.
 
 <Row>
 <Column colLg={8}>
 
-![Example of a standard creation modal](images/2.png)
+![Example of a standard creation modal.](images/2.png)
 
-<Caption>Example of a standard creation modal</Caption>
+<Caption>Example of a standard creation modal.</Caption>
 
 </Column>
 </Row>
 
 ### Side panel
 
-If you have limited screen real estate, consider using a side panel. Side panels keep the user aware of the context in which they are working. You may choose to use either slide in or slide over panels.
+Consider using a side panel to take advantage of the added real-estate and keep the user in context of where they were working. You may choose to use either the slide in or slide over panels.
 
 <Row>
 <Column colLg={8}>
 
-![Example of a side panel creation flow](images/3.png)
+![Example of a side panel creation flow.](images/3.png)
 
-<Caption>Example of a side panel creation flow</Caption>
+<Caption>Example of a side panel creation flow.</Caption>
 
 </Column>
 </Row>
 
-### Full page
+### Full screen
 
-If your creation flow consists of multiple fields and/or supporting imagery, consider the full-page option. Full-page creation flows allow you to take advantage of maximum screen real estate.
+If your creation flow consists of multiple fields and/or supporting imagery, consider the full screen option to take full advantage of the screen’s available real estate.
 
 <Row>
 <Column colLg={8}>
 
-![Example of a full-page creation flow](images/4.png)
+![Example of a full-page creation flow.](images/4.png)
+
+<Caption> Example of a full page creation flow. </Caption>
 
 </Column>
 </Row>
 
 <Row>
 <Column colLg={8}>
 
-![Example of a full-page creation flow](images/5.png)
+![Another example of a full-page creation flow.](images/5.png)
 
-<Caption>Example of a full-page creation flow</Caption>
+<Caption>Another example of a full-page creation flow.</Caption>
 
 </Column>
 </Row>
 
 ## Multistep creation
 
-If the creation task is complex, consider breaking the process up into multiple steps. Display related or dependent options together.
+If the creation task is complex, consider whether breaking the process up into steps could help the user. Consider displaying related or dependant options together.
 
-Multistep creation follows the same principles as standard creation but the creation takes place through several steps. This flow can take place in a side panel or can use the full display.
+Multistep creation follows the same principles as [Standard creation](#standard-creation), with the addition of multisteps.
+
+### Side panel
 
-### Side panel multistep
+Consider using a side panel to take advantage of the added real-estate and keep the user in context of where they were working. You may choose to use either the slide in or slide over panels.
```

Comment:
```suggestion
Consider using a side panel to take advantage of the added real estate and keep the user in the context of where they are working. You may choose to use either the slide in or slide over panels.
```

Reply:
I think we referred to "screen real estate" above, can we use that phrase again here for consistency.

Reply:
Absolutely! I was just removing the hyphen.